src/data/clients.ts

```diff
@@ -719,6 +719,22 @@ export const Clients: Array<Client> = [
       }
     ]
   },
+  {
```

Comment:
New additions should be added to the bottom of the list

Reply:
I think I agree will @thornbill's ordering.

Reply:
Yep should be fine considering it was already discussed 👍